tools/autodeployment/server.js

```diff
@@ -27,9 +32,9 @@ const credentials = {
   cert: certificate,
 };
 
-function handleError(req, res) {
+function handleError(req, res, error = 'Not Found') {
```

Comment:
@manekenpix 
we are using ES6 in almost every file (i.e arrow functions) could we use it here too? Just a question.

Reply:
I know, I just wanted to be consistent with the way this file was written, otherwise I'd have to change everything (and this PR is not about switching to ES6).